src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1264,11 +1127,6 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand, tar_name):
                     'site:%s/%s > %s' % (session_sandbox, sdist, sdist)
                 ])
 
-            if stage_cacerts:
-                jd.file_transfer.extend([
-                    'site:%s/%s > %s' % (session_sandbox, certs, certs)
-                ])
```

Comment:
two interdependent questions: why this was deleted? and is section under `if not shared_filesystem` actually needed?

Reply:
@andre-merzky can you also please look here ? (from initial changes as well)

Reply:
Thanks for the ping!  We don't have any systems where this is still needed, and, more importantly, pip distribution was fixed to not need this anymore in recent releases.  The assumption here is that no system will install an older version of pip anymore.
The underlying problem was that some pip installations did not manage to fetch pypi host certs during installation and thus could not fetch from pypi...

Reply:
the check for `shared_filesystem` is needed for condor.  Well, I don't have any idea in what state our condor support is really as we have not exercised this in the last 5 years or so, but still.
Condor compute nodes have no shared file system with the condor login nodes (not sure about node type terminology, but you get the point), and we thus need to stage in two steps: once from the client side to the login node, and once from the login node to the compute node (each compute node in Condor runs it's own agent IIRC).  That is not needed on clusters which have a shared fs.

Reply:
`cacerts`: good catch, will clean up.

shared FS: right, good point.  Probably would be good to fix even if we can't test - let me do that.

Reply:
Cleaned it out...